Utils/VersionBump/main.swift

```diff
@@ -43,9 +35,7 @@ if let match = Regex(semver, options: [.dotMatchesLineSeparators]).firstMatch(in
     fromVersion = extractVersionOnly(fromVersion)
     toVersion = extractVersionOnly(toVersion)
     
-    for file in restrictFiles {
-        try updateVersion(file, fromVersion, toVersion)
-    }
+    try updateVersion("./Sources/Configuration/Versioning.xcconfig", fromVersion, toVersion)
```

Comment:
`h`: It did change some files:
<img width="383" alt="Screenshot 2025-05-07 at 13 22 21" src="https://github.com/user-attachments/assets/a5a9004c-a8ac-4e2a-bbd8-dcd527b8f8b8" />

It did not update these files:
- `Sources/Configuration/Versioning.xcconfig`
- `Samples/Shared/Config/Versioning.xcconfig`
- `Samples/SessionReplay-CameraTest/config/Versioning.xcconfig` (not sure if this one is expected)